guides/configuration/drupal_application.md

```diff
@@ -29,4 +31,4 @@ crons:
 
 In this example, we are only using the ``mysql`` service. You can use more services (redis, solr...) if you need.
 
-We are also supposing that your Drupal codebase is at the root of your Git repository. If it's not the case, you can change the ``document_root`` to the proper folder where your Drupal files are located.
\ No newline at end of file
+This example presumes that your Drupal codebase is in the "web" directory of your repository. If it's not the case, you can change the ``root`` to the proper folder where your Drupal files are located. Note that putting your Drupal webroot at the root of the repository is not supported.
```

Comment:
Usually, I assume it's the former if something that happens to work is described in the documentation as not supported.